File: Objetivo 1/Objetivo1_Exercicios_PHP/model/Conta.php
<?php

/**
 * Description of Conta
 *
 * @author vagner
 */
class Conta {
    private $saldo;
    
    public function __construct() {}
    
    public function getSaldo() {
        return $this->saldo;
    }

   
    public function deposita($valor){
        $this->saldo += $valor;
     }
    
    public function saca($valor){
        $this->saldo -= $valor;
    }
 
    
    public function atualiza($taxa){
        $this->saldo *= ($taxa/100);
    }

    public function __toString() {
        return "Conta[saldo=$this->saldo]";
    }

}

File: Objetivo 1/Objetivo1_Exercicios_Java/src/model/Conta.java
package model;

/**
 *
 * @author vagner
 */
public class Conta {
    private double saldo;

    public Conta(){}
    
    public Conta(double valor) {
        this.saldo = valor;
    }
     
    public double getSaldo(){
        return this.saldo;
    }
    
    public void deposita(double valor){
        this.saldo += valor;
     }
    
    public void saca(double valor){
        this.saldo -= valor;
    }
 
    
    public void atualiza(double taxa){
        this.saldo *= (taxa/100);
    }

    @Override
    public String toString() {
        return "Conta{" + "saldo=" + saldo + '}';
    }
        
}
